script.js

```diff
@@ -0,0 +1,54 @@
+let USER_INPUT_ELEMENT = document.getElementById("date")
+userInput.max = new Date().toISOString().split("T")[0];
+let _RESULT_ELEMENT = document.getElementById("result");
```

Comment:
instead of _RESULT_ELEMENT just RESULT_ELEMENT. Remove this extra leading underscore. it is sometimes used to show that this variable is private in some languages that do not have access modifiers (for example python).

Reply:
done